orchestrator-salt/src/main/resources/salt-common/salt/fluent/template/check_fluent_plugins.sh.j2

```diff
@@ -64,13 +64,17 @@ function check_and_install_plugin() {
 }
 
 function main() {
+{% if not fluent.fluent_prewarm_role in grains.get('roles', []) %}
```

Comment:
no need extra check for the script if there is a check around creating this template + executing it

Reply:
Will remove this.